audio-worklet/basic/bit-crusher.html

```diff
@@ -2,16 +2,29 @@
 <html>
   <head>
     <title>
-      AudioWorklet: BitCrusher | Chrome WebAudio Samples
+      AudioWorklet: BitCrusher with AudioParam Automation | Chrome WebAudio Samples
     </title>
     <link rel="stylesheet" href="../../resources/base.css">
     <script src="../lib/audio-worklet-helper.js"></script>
   </head>
   <body>
     <a href="../index.html" class="link-small">&#171; AudioWorklet home</a>
-    <h1>
-      Demo: BitCrusher
-    </h1>
+    <section>
+      <h1>
+        Demo: BitCrusher with AudioParam Automation
+      </h1>
+      <dt>A BitCrusher example from the specification, but modified to
+      demonstrate AudioParam automations. The demo runs for 8 seconds.</dt>
+      <dd>
```

Comment:
Maybe say the source is an oscillator and say the kinds of automation if there are only a couple.

Reply:
The kinds of automation can be found with a single click 1 inch below. :)
I'll clarify the source.